### DISCUSSION TITLE
Habits folder

### DISCUSSION THREAD

**Discussion by Dzoke69** [iSoron/uhabits#1924] [Feature Requests]
I always wanted to have this feature on this app. A few times I wanted to uninstall the app but really love this app. I want like a dropdown menu for habits like "skincare" and when i click on it, it opens more options like cleanser, toner etc. If you understand I would love this feature

**Comment by SparkDeni** [comment]
The app is very good, I posted this request on the play store today.  

1. Group habits into groups, so you only see specific groups of habits that are related to each other. 
2. Another suggestion is to include the possibility of data synchronization in the cloud using a login and password.

**Comment by gelatinbomb** [comment]
I would like that feature too. Something simple that would allow to group some habits like exercise routines for example. Is reall y useful especially if you have a lot of habits.

**Comment by inson1** [comment]
@Dzoke69 Isnt this duplicate of https://github.com/iSoron/uhabits/discussions/495?
if yes please close it :)
